Create a makefile to install the pre-built binaries and headers etc.
--- Makefile	1970-01-01 01:00:00.000000000 +0100
+++ Makefile	2016-11-10 17:31:08.686722661 +0000
@@ -0,0 +1,27 @@
+all:	libqhyccd.pc
+
+%.pc:	%.pc.in
+	sed -e "s!@LIBDIR@!$(LIBDIR)!g" \
+		-e "s!@VERSION@!$(VERSION)!g" < $< > $@
+
+install:
+	mkdir -p $(LIBDIR) $(INCDIR) $(PKGCONFDIR) $(UDEVDIR)
+	case $(DEB_BUILD_ARCH) in \
+	i386) \
+		cp $(SRCDIR)/usr/local/lib/libqhyccd.a $(LIBDIR); \
+		cp $(SRCDIR)/usr/local/lib/libqhyccd*.so.$(VERSION) $(LIBDIR);; \
+	amd64) \
+		cp $(SRCDIR)/usr/local/lib/x64/libqhyccd.a $(LIBDIR); \
+		cp $(SRCDIR)/usr/local/lib/x64/libqhyccd*.so.$(VERSION) $(LIBDIR);; \
+	arm64) \
+		cp $(SRCDIR)/usr/local/lib/armv8/libqhyccd.a $(LIBDIR); \
+		cp $(SRCDIR)/usr/local/lib/armv8/libqhyccd*.so.$(VERSION) $(LIBDIR);; \
+	armhf) \
+		arch=`uname -m | sed -e 's/[lb]$$//'`; \
+		cp $(SRCDIR)/usr/local/lib/armv7/libqhyccd.a $(LIBDIR); \
+		cp $(SRCDIR)/usr/local/lib/armv7/libqhyccd*.so.$(VERSION) $(LIBDIR);; \
+	esac
+	ln -sf libqhyccd.so.$(VERSION) $(LIBDIR)/libqhyccd.so.$(MAJOR_REV)
+	ln -sf libqhyccd.so.$(VERSION) $(LIBDIR)/libqhyccd.so
+	cp $(SRCDIR)/usr/local/include/*.h $(INCDIR)
+	cp $(SRCDIR)/libqhyccd.pc $(PKGCONFDIR)
